templates/keycloak-provision.sh

```diff
@@ -42,7 +42,7 @@ provisionKeycloak() {
     -s clientId={{ .KeycloakClientId }} \
     -s id={{ .KeycloakClientId }} \
     -s webOrigins='["http://{{ .CheHost }}", "https://{{ .CheHost }}"]' \
-    -s redirectUris='["http://{{ .CheHost }}/dashboard/*", "https://{{ .CheHost }}/dashboard/*", "http://{{ .CheHost }}/workspace-loader/*", "https://{{ .CheHost }}/workspace-loader/*", "http://{{ .CheHost }}/_app/*", "https://{{ .CheHost }}/_app/*", "http://{{ .CheHost }}/swagger/*", "https://{{ .CheHost }}/swagger/*"]' \
```

Comment:
Why do we remove `workspace-loader` ? Is it not used anymore?

Reply:
It is not exist anymore AFAIK.

Reply:
Who can confirm that?

Reply:
I can: https://github.com/eclipse/che/pull/18970